### PR TITLE
Add missing page title for published screen

### PR DESCRIPTION
**Pull request by benthorner**
Content taken from the prototype here:

https://content-publisher-prototype.cloudapps.digital/content-summary-published

### CODE REVIEW

**Comment by benthorner** [comment]
@tijmenb ah, I was thinking about the browser tab specifically, which just has '- GOV.UK'. Shall I fix this in the layout instead? (Currently it re-uses the page title without any alternative.)

**Comment by tijmenb** [comment]
@benthorner ah! Yeah maybe an extra option to the layout?

```rb
<% content_for :browser_title, t("publish_document.published.title") %>
```

**Comment by benthorner** [comment]
@tijmenb updated :-).

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 <%= render 'govuk_publishing_components/components/layout_for_admin',
   environment: GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment,
-  browser_title: yield(:title) do %>
+  browser_title: (yield(:browser_title).presence || yield(:title)) + " - GOV.UK" do %>
```
Comment (tijmenb) [inline app/views/layouts/application.html.erb]: `" - GOV.UK" ` was removed in a previous commit, as it's now added by the component.

Comment (benthorner) [reply app/views/layouts/application.html.erb]: Ah, sorry - didn't reload my rails. That's better!

**Pull request merged**